src/solvers/flattening/boolbv.cpp

```diff
@@ -632,12 +632,8 @@ bool boolbvt::is_unbounded_array(const typet &type) const
 void boolbvt::print_assignment(std::ostream &out) const
 {
   arrayst::print_assignment(out);
-
-  for(const auto &it : map.mapping)
-  {
-    out << it.first << "="
-        << it.second.get_value(prop) << '\n';
-  }
+  for(const auto &pair : map.mapping)
+    out << pair.first << "=" << pair.second.get_value(prop) << '\n';
```

Comment:
The commit message doesn't really reflect what happened here (which mainly is renaming it -> pair (which is good).

Reply:
I changed the commit message: https://github.com/diffblue/cbmc/pull/1810/commits/d400debc5e565c7f37ff7cd57efca9bb5571340f